app/components/empire-spell.js

```diff
@@ -0,0 +1,90 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  empire: undefined,
+  tagName: 'span',
```

Comment:
Of course, there is the issue of making a `span` to put a `div` into it…
I will make a future issue of this, as I am quite tired of this spell nonsense